content/md/first-steps/manage-multidimensional-data-in-a-table.md

```diff
@@ -132,7 +132,17 @@ And if you are comfortable with the API, you can retrieve the table options with
 ## Import/export the table values of a product
 Exactly like for other attribute types, you can import and export your products with table attributes. But, as a table is a quite complexe value, the only way for the PIM to import table data among the other attributes of the product, is by translating your value into JSON in the "table" attribute column of your CSV or XLSX file.
 
-That's why **we highly recommend you to keep the import/export feature for data flows between the PIM and other systems purpose, and not for human enrichment purpose.**
+### Use the dedicated "product data table values" import/export jobs
+We know that, working with JSON format within a xlsx/csv file is not that much comfortable for manual enrichment. That's why we have created several import and export jobs in order to import and export the enriched table attributes, formatted as they are in the product edit form.  
+
+These dedicated job are named:
+- `Product table values` import and export (csv and xlsx)
+- `Product model table values` import and export (csv and xlsx)
+
+You should create **1 import profile per table attribute** you want to import/export. Indeed, these jobs enable you to import/export the table values of several products sharing the same table attribute. As each table attribute can have a different structure, we cannot mix them up in the same file :wink:.
```

Comment:
In fact they can export values only attached to 1 table attribute, but they can import a file with different attributes. Maybe it's a very specific usecase that don't deserve to be documented...

Reply:
```suggestion
We recommend you to create **one import profile per table attribute** you want to import/export. Indeed, these jobs enable you to import/export the table values of several products sharing the same table attribute. For imports, as each table attribute can have a different structure, we don’t want to import files mixing up tables with different structures. For exports, you need to choose a single attribute for each job.
```